### DISCUSSION TITLE
swizzin works OK with OVH VPS, not with AWS Lightsail !

### DISCUSSION THREAD

**Discussion by siliconhippy** [swizzin/swizzin#1031] [Q&A + Help] [answered]
1. Swizzin works nicely with OVH VPS, with 2GB RAM, 1 vCPU and 40 GB HDD, with Ubuntu 20.04.
2. However, when I set up swizzin using the script and add Jellyfin , panel, webmin, ngnix, rclone, duckdns to the program install list, everything looks fine as installed. 
After a few moments, the SSH access goes blank.
I have tried this a dozen times over weeks with 3-4 location changes, with the $10 (2 GB RAM, 1vcpu, 60 GB SSSD), Ubuntu 22.04, and less instances. Same result...any clues?

#ufw status shows blank. By the time I try saying #ufw allow, the terminal has already frozen, and reboot is the same. Under IPv4 Firewall, I added 22, 80, 443, 53 (DNS), icmp ping, and 8333 (swizzin panel) but no change.

I cannot access the swizzin panel either via http://public-IP-instance.
Asked AWS support multiple times- they don't see the problem.

**Comment by flying-sausages** [comment]
Try installing one thing at a time and see when it stops responding? Also keep an eye out on `htop` or something similar so you can see if it's something like RAM being clogged up or something. If you've got KVM access, keep that open and see what it do when `ssh` stops working. Best of luck!

**Comment by siliconhippy** [comment, marked as answer]
It was a problem with AWS Lightsail....stops SSH access if you use # ufw !
Not using ufw but only using AWS Networking firewall settings solved problems with swizzin and other installs ☺️
